package.json

```diff
@@ -65,7 +65,7 @@
     "babylon": "^6.5.0",
     "commitizen": "^2.9.6",
     "cz-conventional-changelog": "^2.0.0",
-    "eslint": "^4.3.0",
```

Comment:
Shouldn't you be able to keep the `^` for `eslint` and `prettier`? (Shouldn't be any breaking changes unless they are not semver compliant, right?) Does the CI build pass if you just keep the `chalk@2` resolution below?

Reply:
Nope, here what happens when eslint gets an upgrade: https://circleci.com/gh/yarnpkg/yarn/19147?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

And for Prettier, well try it yourself, it reports about 1,400 errors because the defaults evolved. 

As stated in https://github.com/yarnpkg/yarn/pull/6208#issuecomment-410082800, I think those problems should be addressed in a separate PR.